rpm-build/SPECS/securedrop-workstation-dom0-config.spec

```diff
@@ -124,10 +124,13 @@ find /srv/salt -maxdepth 1 -type f -iname '*.top' \
     | xargs qubesctl top.enable > /dev/null
 
 # Force full run of all Salt states - uncomment in release branch
-# mkdir -p /tmp/sdw-migrations
-# touch /tmp/sdw-migrations/fedora-39-update
+mkdir -p /tmp/sdw-migrations
+touch /tmp/sdw-migrations/fedora-39-update
```

Comment:
The existence of this mechanism at the packaging level feels to me like a symptom of the need for <https://github.com/freedomofpress/securedrop-workstation/issues/966#issuecomment-2035435574>.  That said: I'm glad we have it right now!